features/admin.userstores.v1/components/edit/edit-connection-details-user-store.tsx

```diff
@@ -82,7 +84,8 @@ export const EditConnectionDetails: FunctionComponent<EditConnectionDetailsProps
         readOnly,
         type,
         update,
-        [ "data-testid" ]: testId
+        [ "data-testid" ]: testId,
+        userStore
```

Comment:
Let's keep custom attributes last.

```suggestion
        userStore,
        [ "data-testid" ]: testId
```

Reply:
Addressed in commit: [ac30550](https://github.com/wso2/identity-apps/pull/8081/commits/ac30550f78156ddb2c0aaf8b57b2a5f840b9226b)